docs/layers.md

```diff
@@ -88,17 +88,17 @@ Mapnik supports geometry filtering via the special `mapnik::geometry_type` keywo
 
 #### Data updates
 
-Most Mapzen vector tile content is updated minutely from OpenStreetMap. Low and mid-zoom tiles are updated approximately monthly. Some source data rarely updates – Natural Earth updates approximately yearly.
+Most Tilezen vector tile content is updated minutely from OpenStreetMap. Low and mid-zoom tiles are updated approximately monthly. Some source data rarely updates – Natural Earth updates approximately yearly.
 
 #### Changelog
 
-The current version of Mapzen vector tile "Tilezen" data model is **v1.2.0**.
+The current version of Tilezen vector tile data schema is **v1.5.0**.
 
-Tiles are still in active development, but Mapzen promises to minimize backwards incompatible breaking changes. Data model promises are listed in the Tilezen [SEMANTIC VERSIONING](https://github.com/mapzen/vector-datasource/tree/master/SEMANTIC-VERSIONING.md) statement.
+Tiles are still in active development, but Tilezen promises to minimize backwards incompatible breaking changes. Data model promises are listed in the Tilezen [SEMANTIC VERSIONING](https://github.com/mapzen/vector-datasource/tree/master/SEMANTIC-VERSIONING.md) statement.
 
 If you're signed up for a [Mapzen API key](https://mapzen.com/developers) you should receive an email notifying you of upcoming changes before they are rolled out to production. You can also send your feedback at hello@mapzen.com!
```

Comment:
Should we remove or caveat this paragraph? I don't think `hello@mapzen.com` works any more!

Reply:
Fixed in 9b8b6f9cf4cb0c705c9ad74786a9a2b93f6fbf37 by adding nextzen.org link, and link to Gitter chat room.